File: bench/b01_hello.c
/* b01_hello.c — 最小 hello world 程序，用于二进制体积对比。
 * 不调用 printf，仅返回 42，测纯运行时体积。
 * 与 bench/b01_hello.x / .zig 三语言同语义。无防常量折叠（测体积）。 */

int main(void) {
  return 42;
}

File: bench/b01_hello.zig
// b01_hello.zig — 与 bench/b01_hello.x / .c 等价的 Zig 参照
// 最小 hello world 程序，用于二进制体积对比。
// 使用 pub fn main() void（非 !void）+ std.process.exit 绕过默认错误处理
// 运行时初始化开销，最小化二进制体积。
const std = @import("std");

pub fn main() void {
    std.process.exit(42);
}
